Library/Homebrew/cmd/help.rb

```diff
@@ -74,12 +74,20 @@ def help(cmd = nil, flags = {})
   end
 
   def command_help(path)
+    if Pathname(path).fnmatch?("*/Homebrew/cmd/*")
+      cmd = Pathname(path).basename(".rb")
+      class_name = "#{cmd.to_s.capitalize}Command"
+      if Homebrew.const_defined?(class_name)
+        return command_help_cmd(cmd)
+      end
+    end
+
     help_lines = path.read.lines.grep(/^#:/)
     if help_lines.empty?
       opoo "No help text in: #{path}" if ARGV.homebrew_developer?
       HOMEBREW_HELP
     else
-      help_lines.map do |line|
+      help_lines.map! do |line|
```

Comment:
Confused by this change; if you're doing an inline modification why the `.join.strip` afterwards?